ui/js/controllers/perf/compare.js

```diff
@@ -20,126 +18,6 @@ import { getCounterMap, getInterval, validateQueryParams, getResultsMap,
 import { getApiUrl } from '../../../helpers/url';
 import { getData } from '../../../helpers/http';
 
-perf.controller('CompareChooserCtrl', [
-    '$state', '$stateParams', '$scope', '$q',
-    'localStorageService',
```

Comment:
I think this removes the last usage of `localStorageService` and so the `angular-local-storage` references in `perf.js` and `package.json` can be removed too?

(If the new implementation wants to use local storage, it can use the native `localStorage` API)

Reply:
The usage here of `localStorageService` doesn't actually appear to be doing what it's intended to do: set the query params on submit with the last selection and then on next page load, utilizing anything stored (among other options) so doing away with it altogether.